samples/GuestConfiguration/deployextension-installed-application-linux/README.md

```diff
@@ -0,0 +1,48 @@
+# Deploy extension to audit for installed applications - Linux
+
+This policy ensures that the requirements are deployed to audit a VM for applications that must be installed.  The requirements include a guest assignment, a managed service identity, and the guest configuration extension.
+
+This policy requires another definition to also be deployed that audits the results published in Azure Resource Manager.  Please see the policy sample
+[Audit for installed application - Linux](..\audit-installed-application-linx\README.md).
+
+## Azure Policy Docs
+
+See more information and a complete walk-through of using this sample on
+[docs.microsoft.com](https://docs.microsoft.com/azure/governance/policy/samples/deployextension-installed-application-linux).
+
+## Try with Azure portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/?#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FGuestConfiguration%2Fdeployextension-installed-application-linux%2Fazurepolicy.json)
+[![Deploy to Azure Gov](https://docs.microsoft.com/azure/governance/policy/media/deploy/deployGovbutton.png)](https://portal.azure.us/?#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FGuestConfiguration%2Fdeployextension-installed-application-linux%2Fazurepolicy.json)
+
+## Try with Azure PowerShell
+
+````powershell
+# Create the Policy Definition (Subscription scope)
+$definition = New-AzPolicyDefinition -Name 'deployextension-installed-application-linux' -DisplayName 'Audit that an application is installed inside Linux VMs' -description 'This policy governs the Audit that an application is installed inside Linux VMs' -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/GuestConfiguration/deployextension-installed-application-linux/azurepolicy.rules.json' -Parameter 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/GuestConfiguration/deployextension-installed-application-linux/azurepolicy.parameters.json' -Mode All
```

Comment:
-mode Indexed for both PS and CLI on this one

Reply:
fixed